colorcet/__init__.py

```diff
@@ -66,12 +67,12 @@
 
 class AttrODict(OrderedDict):
     """Ordered dictionary with attribute access (e.g. for tab completion)"""
-    def __dir__(self): return self.keys()
-    def __delattr__(self, name): del self[name]
-    def __getattr__(self, name):
-        return self[name] if not name.startswith('_') else super(AttrODict, self).__getattr__(name)
-    def __setattr__(self, name, value):
-        if (name.startswith('_')): return super(AttrODict, self).__setattr__(name, value)
+    def __dir__(self) -> List[str]: return list(self.keys())
+    def __delattr__(self, name: str) -> None: del self[name]
```

Comment:
I think it is OK to move these down a line, with type hinting, it becomes very hard to read these one-liners. 
```suggestion
    def __delattr__(self, name: str) -> None: 
        del self[name]
```

Reply:
i agree. I did it like that before, then reverted it.